### ISSUE TITLE
Where the question ids come from?

### ISSUE THREAD

**Issue by jpdonaghy** [jpdonaghy/angular-dynamic-forms#9]
The meta-data doesn't contain id properties, will we be auto generating something, configuring them or some other convention?

**Comment by thelgevold** [comment]
It might not be the best name, but some of the sample metas I've seen have a "questionId". However, if that's not required/consistent we should come up with some alternatives.

**Comment by jpdonaghy** [comment]
That meta is a very specific use case and is currently the 1%.

**Comment by thelgevold** [comment]
Yeah, it's basically just "something" unique about a question. I've seen other metas use unique property names for each question. We can generate a pseudo key here - perhaps based on the index of the question of similar.
